sahi/model.py

```diff
@@ -1080,6 +1080,13 @@ def load_model(self):
                 "Check 'https://tfhub.dev/tensorflow/collections/object_detection/' for supported TF Hub models."
             )
 
+        if self.category_mapping is None:
+            from sahi.utils.tensorflow import COCO_CLASSES
+
+            category_mapping = {str(i): COCO_CLASSES[i] for i in range(len(COCO_CLASSES))}
+
+        self.category_mapping = category_mapping
```

Comment:
@kadirnar bu satirin if'in icinde alinmasi gerekiyor yoksa if self.category_mapping None olmadigi zaman hata verir

Reply:
Evet çok doğru. Önceki satırlarda o işlem olduğunu düşünerek if'in dışına almıştım. Düzelttim.